tests/python/shared/assets/cvat_db/data.json

```diff
@@ -394,7 +394,7 @@
   "pk": 18,
   "fields": {
     "password": "md5$rKclttm53lRHC0rITF48uu$7ac59be88ed2a5de4b513edbef629258",
-    "last_login": "2022-03-05T09:37:01.636Z",
+    "last_login": "2024-07-15T15:27:57.390Z",
```

Comment:
Typically we clean these changes manually, if they are not mandatory to test the update.

Reply:
cleaned these changes